Projects/TravelCertificate/Sources/Store/TravelInsuranceAction.swift

```diff
@@ -16,16 +16,16 @@ enum TravelInsuranceAction: ActionProtocol, Hashable {
     case setDate(value: Date, type: TravelInsuranceDatePickerType)
     case toogleMyselfAsInsured
     case setPolicyCoInsured(PolicyCoinsuredPersonModel)
-    case updatePolicyCoInsured(PolicyCoinsuredPersonModel, with: PolicyCoinsuredPersonModel)
-    case removePolicyCoInsured(PolicyCoinsuredPersonModel)
     case setDownloadUrl(urL: URL)
 
     case navigation(TravelInsuranceNavigationAction)
     case getTravelCertificateSpecification
     case travelCertificateSpecificationSet
+    case goToEditCoInsured
+    case dismissTravelInsuranceFlow
 }
 
-enum TravelInsuranceNavigationAction: ActionProtocol, Hashable {
+public enum TravelInsuranceNavigationAction: ActionProtocol, Hashable {
```

Comment:
openCoinsured
dismissAddUpdateCoinsured

goToEditCoInsured
can be removed I think

Reply:
@sladan-hedvig goToEditCoInsured should not be deleted since I just added that one.  It's the deep link to edit co-insured.

Removed the others.